frontend/src/components/maps/leaflet/InventoryLayer.tsx

```diff
@@ -80,6 +80,12 @@ export const InventoryLayer: React.FC<InventoryLayerProps> = ({
       maxLandArea: filter?.maxLandArea,
       inSurplusPropertyProgram: filter?.inSurplusPropertyProgram,
```

Comment:
With so many properties it feels like it should just accept the filter object itself instead.